### ISSUE TITLE
Feature request: generated column support

### ISSUE THREAD

**Issue by zopieux** [qwtel/sqlite-viewer-vscode#28]
```sql
CREATE TABLE foo (
real_column TEXT,
gen_column NUMERIC AS (json_extract(real_column, '$.foo'))
);
INSERT INTO foo (real_column) VALUES ('{"foo":42}');
SELECT * FROM foo;
```

This is valid SQLite, but the viewer only displays the “physical” columns and not the [“generated”](https://www.sqlite.org/gencol.html) ones. They should be searchable, sortable, etc. just like physical columns, with the obvious exception that they are read-only and other limitations described in the linked page. Note generated columns can be either *virtual* or *stored*.

**Comment by qwtel** [comment]
Interesting. The UI is currenlty baesd on pragma_table_info, it might be that generated columsn aren't in there. I have to move awayt from that model anyway to make joins, views, and query runner work.

**Comment by qwtel** [comment]
Actually, this was a one character fix. SQLite has a `pragma_table_xinfo` table that includes generated columns. Shipped as v0.4.14.
